cmd/apps/certmanager_app.go

```diff
@@ -45,7 +45,7 @@ func MakeInstallCertManager() *cobra.Command {
 			return err
 		}
 
-		clientArch, clientOS := env.GetClientArch()
+		clientArch, clientOS := env.GetClientArch(true)
```

Comment:
What does true do? It's not documented and looks confusing

Reply:
Added a bool parameter streamStdio to function GetClientArch in pkg/env. Currently StreamStdio is hardcored as true. Running app minio info will print out clientArch and clientOS.
Hence, I added this to suppress the unwanted output.

Where's the best place to document this change?

Reply:
As a contributor, I would personally ask the maintainer if this change is wanted, rather than assuming it.

I would rather not take this change in the PR, or just force it always to be silent, you pick, but the `bool true` is confusing and doesn't feel necessary.